src/App.js

```diff
@@ -1,22 +1,33 @@
-import React, { Component } from 'react';
-import './App.css';
+import React, { Component } from "react";
+// import storm from "./img/weather-icons/storm.svg";
+// import drizzle from "./img/weather-icons/drizzle.svg";
+// import rain from "./img/weather-icons/rain.svg";
+// import snow from "./img/weather-icons/snow.svg";
+// import partlycloudy from "./img/weather-icons/partlycloudy.svg";
+import clear from "./img/weather-icons/clear.svg";
+import mostlycloudy from "./img/weather-icons/mostlycloudy.svg";
+import Forcast from "./Components/Forcast";
+import SearchBar from "./Components/SearchBar";
+import CurrentWeather from "./Components/CurrentWeather";
 
-class App extends Component {
+import "./App.css";
+import "./grid.css";
+// import SearchBar from "./Components/SearchBar";
+// import CurrentWeather from "./Components/CurrentWeather";
 
+class App extends Component {
   constructor(props) {
     super(props);
-    this.state = {
-    };
+    this.state = {};
   }
 
-  render () {
+  render() {
     return (
-      <div className="app">
-        <header className="app__header">
-        </header>
-        <main className="app__main">
-        </main>
-      </div>
+      <section>
+        <SearchBar buttonTitle="searchCity" />
+        <CurrentWeather />
+        <Forcast time="3:00" imgSrc={mostlycloudy} temprature="8°C" />
```

Comment:
A `Forecast` doesn't have just image or one `temperature`, or one `time`.  I imagine you were just practicing with props here.

Watch your spelling - people will judge you by it: `Forecast` and `temperature`